float64 scale_estimate
bool converged
float32 std_dev
float32 mean
